src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -231,28 +324,66 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
       applyThemesOnElement(this, this.hass.themes, this._config.theme);
     }
 
+    if (!this._progressInterval && this._showProgressBar) {
+      this._progressInterval = window.setInterval(() => {
+        this._getProgressValue();
+      }, 1000);
+    }
```

Comment:
Let's make sure you also clear the interval when it no longer should show a progress bar.